scripts/install_environment.sh

```diff
@@ -13,8 +13,31 @@ function version_is_greater() {
 }
 
 function install_libvirt() {
+    source /etc/os-release  # This should set `PRETTY_NAME` as environment variable
+
+    # RHEL and CentOS require epel-release for swtpm and swtpm-tools packages
+    case "${PRETTY_NAME}" in
+    "Red Hat Enterprise Linux 8"* | "CentOS Linux 8"*)
+        sudo dnf install -y \
+            https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+        ;;
+    "Red Hat Enterprise Linux 7"* | "CentOS Linux 7"*)
+        sudo dnf install -y \
+            https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+        ;;
+    esac
+
     echo "Installing libvirt..."
-    sudo dnf install -y libvirt libvirt-devel libvirt-daemon-kvm qemu-kvm libgcrypt
+    sudo setenforce enforcing && sudo dnf install -y \
+        libvirt \
+        libvirt-devel \
+        libvirt-daemon-kvm \
+        qemu-kvm \
+        libgcrypt \
+        swtpm \
+        swtpm-tools \
+            && sudo setenforce permissive
```

Comment:
Same as https://github.com/openshift/assisted-test-infra/pull/1142/files#r725829881

Reply:
still under investigation, I hope to find a proper solution for the installation part